src/style.css

```diff
@@ -33,3 +33,8 @@ header {
   padding-left: 1rem;
   color: purple;
 }
+
```

Comment:
we are trying to get rid of the `style.css` file, by switching the project to Material UI.

For now, could you just add this inline style in the components instead? `style={{ marginRight: '2rem' }}`

---

We'll need to discuss images as a team. MUI doesn't have an image component, they recommend another library dependency: https://mui.com/material-ui/discover-more/related-projects/#image

Reply:
> we are trying to get rid of the `style.css` file, by switching the project to Material UI.
> 
> For now, could you just add this inline style in the components instead? `style={{ marginRight: '2rem' }}`


This still